frontend/src/metabase/admin/settings/app/components/SettingsEditor/SettingsEditor.jsx

```diff
@@ -88,7 +88,7 @@ class SettingsEditor extends Component {
    * @param {function} [options.onChanged] - callback fired after the setting has been updated
    * @param {function} [options.onError] - callback fired after the setting has failed to update
    */
-  updateSetting = async (setting, newValue, options) => {
+  handleUpdateSetting = async (setting, newValue, options) => {
```

Comment:
the fact that the name of this function was the same as the `updateSetting` completely confused then completely annoyed me, so I changed it

Reply:
this whole component is detined for the fire soon ™️ , because there's nothing about it that doesn't bother me 😁